Firestore/src/Admin/V1/FirestoreAdminClient.php

```diff
@@ -25,10 +25,28 @@
 namespace Google\Cloud\Firestore\Admin\V1;
 
 use Google\Cloud\Firestore\Admin\V1\Gapic\FirestoreAdminGapicClient;
+use Google\ApiCore\PathTemplate;
 
 /** {@inheritdoc} */
 class FirestoreAdminClient extends FirestoreAdminGapicClient
 {
-    // This class is intentionally empty, and is intended to hold manual additions to
-    // the generated {@see FirestoreAdminGapicClient} class.
+    /**
+     * Formats a string containing the fully-qualified path to represent
+     * a parent resource.
+     *
+     * @param string $project
+     * @param string $database
+     * @param string $collectionId
+     *
+     * @return string The formatted parent resource.
+     * @deprecated
+     */
+    public static function parentName($project, $database, $collectionId)
```

Comment:
Should we add a warning(_and possibly an alternative_) like we have done in `BigQueryDataTransfer/src/V1/DataTransferServiceClient.php`?

Seems like that will be helpful.

Reply:
It's a good idea but I am not aware of an alternative, I only know the GAPICs removed them. So I just added `@deprecated` instead

@dwsupplee may know more about this

Reply:
@saranshdhingra white making this PR I looked through the existing methods to see if the same template existed under a different name and couldn't find any matches. So, unfortunately I don't know what to recommend here.

Reply:
A little background:

A while back there was a migration to a new form of GAPIC configs which put more heavy lifting on to proto annotations. With the annotations, there wasn't a solid way to retain the existing path helpers that were defined by the older GAPIC configs. There isn't a good alternative to these helpers, unfortunately. Adding in the deprecated notice notice and removing them once we're ready for a breaking change is a good compromise/path forward IMO.

Reply:
FYI if you look at the `BigQueryDataTransfer` client, you'll see that I did add calls to existing methods (because they were renamed). But the others didn't have matches, so they've been added as you see here.